Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/FirebaseDatabase.kt

```diff
@@ -75,5 +75,18 @@ open class FirebaseDatabase {
             return detail
         }
     }
+
+    class FaqEvent() {
+
+        class Answer() {
+            var photoLink: String? = null
+            var mapLink: String? = null
+            var otherLink: String? = null
+            var answer: String? = null
+        }
+
+        var answers: List<Answer>? = null
```

Comment:
question and answers should also be non-null

Reply:
done